src/main/java/com/epam/wl/dao/OrderDAO.java

```diff
@@ -3,18 +3,84 @@
 import com.epam.wl.entities.Book;
 import com.epam.wl.entities.Order;
 import com.epam.wl.entities.User;
+import com.epam.wl.enums.OrderStatus;
 
+import javax.sql.DataSource;
+import java.sql.*;
+import java.util.ArrayList;
 import java.util.List;
 
-public interface OrderDAO {
-    void create(User user, Book book);
+public class OrderDAO {
+    private DataSource dataSource;
 
-    void setInProgress(int orderID);
+    public OrderDAO(DataSource dataSource) {
+        this.dataSource = dataSource;
+    }
 
-    void close(int orderID);
+    public void create(User user, Book book) {
+        try (Connection connection = dataSource.getConnection()) {
 
-    List<Order> getAll();
+            PreparedStatement preparedStatement = connection.prepareStatement(
+                    "INSERT INTO order (userid, editionid) VALUES(?, ?);");
+            preparedStatement.setInt(1, user.getId());
+            preparedStatement.setInt(2, book.getId());
+            preparedStatement.executeUpdate();
+        } catch (SQLException e) {
+        }
+    }
 
-    List<Order> getNew();
+    public void setInProgress(int orderID) {
```

Comment:
I think it reasonable to create one method setStatus that takes orderId and Status